### PR TITLE
do not pretend that Non-CGB mode also uses video and RAM banks

### PR DESCRIPTION
**Pull request by AlvaroBurnett**
remove unneeded spaces in the table while we are at it.

### CODE REVIEW

**Comment by CasualPokePlayer** [comment]
How is the previous text inaccurate? In Non-CGB mode, you are locked to bank 0 for VRAM and bank 0-1 for WRAM, and cannot bankswitch between the additional banks the CGB has. Also for C000-CFFF, "In CGB mode, bank 0". What? It's bank 0 either way.

**Comment by AlvaroBurnett** [comment]
Non-CGB mode includes the DMG, which didn't split memory in banks like the text suggested.

File: content/Memory_Map.md [hunk 1]
```diff
@@ -3,20 +3,20 @@ The Game Boy has a 16-bit address bus, which is used to address ROM, RAM, and I/
...
-| FE00        | FE9F      | Sprite attribute table ([OAM](#vram-sprite-attribute-table-oam))   | |
-| FEA0        | FEFF      | Not Usable                                                                                       | Nintendo says use of this area is prohibited |
-| FF00        | FF7F      | I/O Registers                                                                                    | |
-| FF80        | FFFE      | High RAM (HRAM)                                                                                  | |
-| FFFF        | FFFF      | [Interrupts](#interrupts) Enable Register (IE)                                         | |
+| **Start**   | **End**   | **Description**                                                  | **Notes**
+|-------------|-----------|------------------------------------------------------------------|----------
+| 0000        | 3FFF      | 16KB ROM bank 00                                                 | From cartridge, usually a fixed bank
+| 4000        | 7FFF      | 16KB ROM Bank 01\~NN                                             | From cartridge, switchable bank via [MB](#memory-bank-controllers) (if any)
```
Comment (ISSOtm) [inline content/Memory_Map.md:9]: While we're here, maybe rename "MB" to "mapper" for clarity?

File: content/Memory_Map.md [hunk 2]
```diff
@@ -3,20 +3,20 @@ The Game Boy has a 16-bit address bus, which is used to address ROM, RAM, and I/
...
+| 8000        | 9FFF      | 8KB Video RAM (VRAM)                                             | In CGB mode, switchable bank 0/1
+| A000        | BFFF      | 8KB External RAM                                                 | In cartridge, switchable bank if any
```
Comment (ISSOtm) [inline content/Memory_Map.md:11]: Should be "**from** cartridge", for consistency.

File: content/Memory_Map.md [hunk 3]
```diff
@@ -3,20 +3,20 @@ The Game Boy has a 16-bit address bus, which is used to address ROM, RAM, and I/
...
+| C000        | CFFF      | 4KB Work RAM (WRAM)                                              | In CGB mode, bank 0
```
Comment (ISSOtm) [inline content/Memory_Map.md:12]: It's not really needed to mention that this is a single bank, is it?

**Review by ISSOtm — CHANGES_REQUESTED** [review]

**Comment by CasualPokePlayer** [comment]
I understood Non-CGB _mode_ to more be defined as the literally DMG compatibility _mode_ set for DMG only ROMs, and didn't include the literal DMG and the like.

File: content/Memory_Map.md (hunk 3 above)
Comment (AlvaroBurnett) [reply content/Memory_Map.md:12]: I wish I could represent the two 4KB pieces as a single 8KB block, but I don't know how.

Comment (AlvaroBurnett) [reply content/Memory_Map.md]: Yep, I agree.

File: content/Memory_Map.md [hunk 4]
```diff
@@ -3,20 +3,20 @@ The Game Boy has a 16-bit address bus, which is used to address ROM, RAM, and I/
...
-| E000        | FDFF      | Mirror of C000\~DDFF (ECHO RAM)                                                                  | Nintendo says use of this area is prohibited. |
-| FE00        | FE9F      | Sprite attribute table ([OAM](#vram-sprite-attribute-table-oam))   | |
-| FEA0        | FEFF      | Not Usable                                                                                       | Nintendo says use of this area is prohibited |
-| FF00        | FF7F      | I/O Registers                                                                                    | |
-| FF80        | FFFE      | High RAM (HRAM)                                                                                  | |
-| FFFF        | FFFF      | [Interrupts](#interrupts) Enable Register (IE)                                         | |
+| **Start**   | **End**   | **Description**                                                  | **Notes**
+|-------------|-----------|------------------------------------------------------------------|----------
+| 0000        | 3FFF      | 16KB ROM bank 00                                                 | From cartridge, usually a fixed bank
```
Comment (ISSOtm) [inline content/Memory_Map.md:8]: [As per the style guide](https://github.com/gbdev/pandocs/wiki/Document-Style#units-and-prefixes), the sizes should be in `KiB`, not `KB`... my bad!

**Review by ISSOtm — CHANGES_REQUESTED** [review]

**Comment by ISSOtm** [comment]
Well, arguably this PR makes this clearer, since it mentions "In CGB mode..." instead of "In Non-CGB mode...".

File: content/Memory_Map.md (hunk 4 above)
Comment (AlvaroBurnett) [reply content/Memory_Map.md]: Fixed

**Review by ISSOtm — APPROVED** [review]

**Pull request merged**